packages/lit-dev-content/site/docs/v3/components/decorators.md

```diff
@@ -44,38 +32,27 @@ The `@customElement` decorator defines a custom element, equivalent to calling:
 customElements.define('my-element', MyElement);
 ```
 
-The `@property` decorator declares a reactive property.
+The `@property` decorator declares a [reactive property](/docs/v3/components/properties/).
 
-See [Reactive properties](/docs/v3/components/properties/) for more information about configuring properties.
+Other decorators provided by the Lit package can be found in the list of [built-in decorators](#built-in-decorators).
```

Comment:
I think we should maybe just move the list of decorators back up here. ie, practical knowledge before theory and configuration.

We should also maybe mention that the decorators are documented along their respective features - or maybe the table does that well enough.

Reply:
My thinking was the list of decorators was a jarring break between discussion of experimental/standard decorators (what i've labeled as "Background") and the "Enabling decorators" section. If we move the "Background"/"Decorator versions and standards" section to be after the "Using decorators" section, then I can get onboard with that.

Agreed that theory can come later, but I was thinking configuration should precede usage. But perhaps I'm thinking too much as a brand new user who's setting everything up. For new users to an established project, the usage is more important and they wouldn't care about usage.

The biggest concern I have for moving the "Background" to be lower is that I added the sentence stating all of our decorator examples assume the usage of experimental decorators, and wanted to a place to link down to standard decorator info/config/usage down below sooner rather than later.

Reply:
Could we do a short note about our example code up top that points to the more detailed background section?